vite.config.ts

```diff
@@ -26,6 +33,21 @@ export default defineConfig({
   optimizeDeps: {
     include: ["./app/routes/**/*"],
   },
+  build: {
+    target: "ES2022",
+    assetsDir: `file-assets`,
+    rollupOptions: {
+      output: {
+        entryFileNames: `file-assets/${buildHash}/[name]-[hash].js`,
+        chunkFileNames() {
+          return `file-assets/${buildHash}/[name]-[hash].js`;
+        },
+        assetFileNames() {
+          return `file-assets/${buildHash}/[name][extname]`;
+        },
```

Comment:
If changing the `assetsDir` is enough to solve the Sentry issue, this could be removed.
AFAIK there is no error handling on the static file middleware that serves these files.

Reply:
Yeah I know, but we have the hash already so I see no issue with using it here. Do you see any downsides?

Reply:
No downsides ;) If we don’t remember why we did that, we now have a note about it :p